ARTCC/High/Irish Sea CTA

```diff
@@ -0,0 +1,53 @@
+;Irish Sea CTA 1 Class C FL195-FL255
+Irish Sea CTA N054.16.27.000 W004.31.00.000 N054.02.46.000 W003.57.56.000
+Irish Sea CTA N054.02.46.000 W003.57.56.000 N053.55.56.000 W004.06.08.000
+Irish Sea CTA N053.55.56.000 W004.06.08.000 N053.54.19.000 W004.21.51.000
+Irish Sea CTA N053.54.19.000 W004.21.51.000 N053.46.17.000 W004.31.12.000
+Irish Sea CTA N053.46.17.000 W004.31.12.000 N053.48.17.000 W005.30.00.000
+Irish Sea CTA N053.48.17.000 W005.30.00.000 N053.52.07.000 W005.30.00.000
+Irish Sea CTA N053.52.07.000 W005.30.00.000 N054.00.05.000 W005.10.32.000
+Irish Sea CTA N054.00.05.000 W005.10.32.000 N054.03.56.000 W005.19.33.000
+Irish Sea CTA N054.03.56.000 W005.19.33.000 N054.02.02.000 W005.05.43.000
+Irish Sea CTA N054.02.02.000 W005.05.43.000 N054.04.05.000 W005.00.40.000
+Irish Sea CTA N054.04.05.000 W005.00.40.000 N053.59.56.000 W004.50.43.000
+Irish Sea CTA N053.59.56.000 W004.50.43.000 N054.16.27.000 W004.31.00.000
+
+;Irish Sea CTA 2 Class C FL195-FL245
+Irish Sea CTA N053.55.56.000 W004.06.08.000 N053.55.22.000 W004.06.49.000
+Irish Sea CTA N053.55.22.000 W004.06.49.000 N053.53.58.000 W004.06.08.000
+Irish Sea CTA N053.53.58.000 W004.06.08.000 N053.41.50.000 W003.36.53.000
+Irish Sea CTA N053.41.50.000 W003.36.53.000 N053.39.18.000 W002.39.00.000
+Irish Sea CTA N053.39.18.000 W002.39.00.000 N053.36.30.000 W003.01.36.000
+Irish Sea CTA N053.36.30.000 W003.01.36.000 N053.32.22.000 W003.00.59.000
+Irish Sea CTA N053.32.22.000 W003.00.59.000 N053.32.13.000 W003.14.06.000
+Irish Sea CTA N053.32.13.000 W003.14.06.000 N053.27.34.000 W003.03.10.000
+Irish Sea CTA N053.27.34.000 W003.03.10.000 N053.23.45.000 W003.16.25.000
```

Comment:
Two lines  with the same ending coordinate - causing an issue in the depiction of the sector.